README.md

```diff
@@ -51,7 +51,7 @@ Submit changes with pull requests.
 
 ### Dependencies ###
 
-1. node & npm
+1. node (version <= 8.4.0) & npm 
```

Comment:
We could also specify the node version in package.json. See https://docs.npmjs.com/files/package.json#engines.